src/components/TodoUser/TodoUser.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import { Todo } from '../../types/Todo.js';
+import cn from 'classnames';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoUser: React.FC<Props> = ({ todo }) => {
+  return (
+    <div
+      data-cy="Todo"
+      className={cn('todo', 'item-enter-done', { completed: todo.completed })}
```

Comment:
use destructurization

Reply:
This comment is not fixed, desetructure all properties